driver/main.c

```diff
@@ -525,7 +525,7 @@ static int ppm_open(struct inode *inode, struct file *filp)
 	consumer->fullcapture_port_range_start = 0;
 	consumer->fullcapture_port_range_end = 0;
 	consumer->statsd_port = PPM_PORT_STATSD;
-	bitmap_fill(consumer->events_mask, PPM_EVENT_MAX); /* Enable all syscall to be passed to userspace */
+	bitmap_fill(consumer->syscalls_mask, SYSCALL_TABLE_SIZE); /* Enable all syscalls to be passed to userspace */
```

Comment:
Why do we need to enable all the syscalls here?

Reply:
Yep this is a default; i don't think it is necessary given that scap_kmod will set all the interesting syscalls at startup time.

Reply:
I'd leave it there for now :)